static/js/reducers/comments.js

```diff
@@ -51,5 +65,13 @@ export const commentsEndpoint = {
       update.set(key, key === postId ? appendCommentToTree(tree, comment, commentId) : tree)
     })
     return update
+  },
+  patchFunc:           (commentId: string, payload: Object) => api.updateComment(commentId, payload),
+  patchSuccessHandler: (response: Comment, data: Map<string, Array<Comment>>) => {
+    let update = new Map()
+    data.forEach((tree, key) => {
+      update.set(key, key === response.post_id ? updateCommentTree(tree, response) : tree)
+    })
```

Comment:
also, it would probably be good to add tests to `reducers/index_test.js` that hit these functions, just to make sure everything is shipshape